Kconfig

```diff
@@ -165,6 +165,12 @@ menu "Camera configuration"
             Maximum value of DMA buffer
             Larger values may fail to allocate due to insufficient contiguous memory blocks, and smaller value may cause DMA interrupt to be too frequent
     
+    config CAMERA_TASK_STACK_SIZE
```

Comment:
Could you put this option next to CAMERA_TASK_PINNED_TO_CORE slightly above, so that the options controlling the task are grouped together?

Reply:
done.